docs/reference/cluster/reroute.asciidoc

```diff
@@ -109,10 +114,13 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
     (Required, object) Defines the commands to perform. Supported commands are:
```

Comment:
GitHub won't allow me to add a comment, but `[role="child_attributes"]` needs to added directly above `==== {api-request-body-title}` or `[[cluster-reroute-api-request-body]]` around line 110.